python/lsst/ts/mtdome/mock_llc/amcs.py

```diff
@@ -379,9 +626,11 @@ async def reset_drives_az(self, start_tai: float, reset: list[int]) -> float:
         Degraded Mode since the drives don't reset themselves.
         The number of values in the reset parameter is not validated.
         """
-        duration = self.azimuth_motion.reset_drives(start_tai, reset)
-        self.end_tai = start_tai + duration
-        return duration
+        for motor_id, val in enumerate(reset):
+            if val == 1:
+                self.drives_in_error_state[motor_id] = False
```

Comment:
Is the `reset` an iterable object such as a `list`? This variable name is a little weird to me if you use the `enumerate()` here.

Reply:
Yes, `reset` is a list. I agree that the name is weird. I'll think of a better one.